content/blog/2018/03/2018-03-20-evolving-mission-of-jenkins.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Evolving Mission of Jenkins"
+tags:
+- jenkins-x
+- essentials
+author: kohsuke
+---
+
+Lately, perhaps subtle but exciting changes are starting to happen in the Jenkins project.
+
+The past few weeks have seen the birth of two new initiatives in Jenkins: https://github.com/jenkinsci/jep/tree/master/jep/300[Jenkins Essentials] and https://github.com/jenkinsci/jep/tree/master/jep/400[Jenkins X].
+Each is exciting in its own right, and I’d encourage interested parties to take a look at their goals and missions and participate in them.
+But in this post, I want to discuss why together these two dots form an important arc, which actually started in the introduction of Jenkins 2 and continued with Blue Ocean.
+
+
+In Jenkins 2, we changed Jenkins so that it starts with richer functionality and more sensible security setup, among other things. 
+This was the first step in a new direction for Jenkins. 
+We changed our focus from “we’ll write plugins and you figure out the rest” to “we’ll write plugins, we’ll assemble them, and we’ll help you be more productive.”
+
+Blue Ocean was another step of this change. 
+We focused on important parts of Jenkins and provided great UX for that. 
+Aside from obvious productivity boost for users, it also decidedly blended together feature areas that are internally provided by a whole bunch of different plugins, but users see much less seam between them.
+
+Jenkins Essentials, which https://github.com/rtyler[Tyler Croy] proposed in recent weeks, is another step forward. 
+That project aims to take an even bigger responsibility in keeping people’s Jenkins instances up and running.
+Like Blue Ocean, Jenkins Essentials focuses on delivering a comprehensive Jenkins usage experience rather than a collection unrelated of plugins which users have to figure out how to wire together.
+It also creates an exciting vehicle for contributors, in which we can develop and deliver features quite differently from how we do it today.
+
+Jenkins X, which was proposed by https://github.com/jstrachan[James Strachan] a few weeks after Jenkins Essentials, is the latest stake on the ground on this same arc.
+Jenkins X brings a different aspect to building a solution — it focuses on a specific vertical area, namely Kubernetes application development, and drastically simplifies the software development in that domain by bringing together Jenkins, a whole bunch of plugins, and an opinionated best practice of how you should use Kubernetes.
+
+
+Collectively, the arc that these efforts form aims to solve the most important and long-heard challenges in Jenkins — ease of use, plugin complexity, fear of upgrade, etc. 
```

Comment:
long-heard challenges? Is this a polite way to phrase "complaints"? Not sure this works as phrased.